markymark/Classes/Styling/HeadingStyling.swift

```diff
@@ -59,6 +64,9 @@ public struct HeadingStyling: ItemStyling, TextColorStylingRule, BaseFontStyling
 private extension Array {
 
     func elementForLevel(_ level:Int) -> Element {
+        if level <= 0 {
+            return self[0]
```

Comment:
consider using the Collection.indices way of checking if your level is outside the boundaries.